glusterd2/peer/self.go

```diff
@@ -47,6 +47,13 @@ func AddSelfDetails() error {
 		PeerAddresses: []string{config.GetString("peeraddress")},
 	}
 
+	if p.MetaData == nil {
+		p.MetaData = make(map[string]string)
+	}
+	if p.MetaData["_zone"] == "" {
+		p.MetaData["_zone"] = "1"
```

Comment:
Do not default it to "1". If zone is not added for any peers then all peers will belong to same zone(that is "1"). Peer.ID can be used as default.

    p.MetaData["_zone"] = p.ID